### PR TITLE
Check if "musicbrainz_albumid" is defined in tags

### PR DESCRIPTION
**Pull request by gentlecat**
*Fixes http://sentry.roman.zone/sentry/ab-production/group/32/.*

### CODE REVIEW

**Comment by Freso** [comment]
Also http://tickets.musicbrainz.org/browse/AB-49

**Comment by gentlecat** [comment]
@Freso, this is going to be fixed by #122.

File: webserver/views/data.py [hunk 1]
```diff
@@ -167,10 +167,11 @@ def _get_recording_info(mbid, metadata):
...
         info['artist_id'] = metadata['tags']['musicbrainz_artistid'][0]
         info['artist'] = metadata['tags']['artist'][0]
-        info['release_id'] = metadata['tags']['musicbrainz_albumid'][0]
+        info['release_id'] = metadata['tags']['musicbrainz_albumid'][0] \
+            if 'musicbrainz_albumid' in metadata['tags'] else None
         info['release'] = metadata['tags']['album'][0]
-        info['track_id'] = metadata['tags']['musicbrainz_releasetrackid'][0] if \
-            'musicbrainz_releasetrackid' in metadata['tags'] else None
+        info['track_id'] = metadata['tags']['musicbrainz_releasetrackid'][0] \
+            if 'musicbrainz_releasetrackid' in metadata['tags'] else None
```
Comment (alastair) [inline webserver/views/data.py]: how about `info['track_id'] = metadata['tags'].get('musicbrainz_releasetrackid', [None])[0]` ?

Alternatively, I'd like to see a helper function which is used for all lines which checks if the field exists (we don't require any tag except recording id to be present) and then does the `[0]`

Comment (gentlecat) [inline webserver/views/data.py]: I like your first idea. Done in 88083614b487c6fecb48116601d9ab893084f7dc.

**Pull request closed without merging**